### PR TITLE
RAB-945: migration DB

### PR DESCRIPTION
**Pull request by MadeWilson**
add a new column to store automation configuration in job_instance table, and an indexed column to check if a job_instance is automated or not. Since we have client with more than 1000+ entry in the table scheduled boolean column is indexed.

**Definition Of Done (for Core Developer only)**

- [x] Tests
- [x] Migration & Installer

### CODE REVIEW

File: upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    private function isColumnAlreadyCreated(string $columnName): bool
+    {
+        $sql = <<<SQL
+            SHOW COLUMNS FROM akeneo_batch_job_instance LIKE :columnName;
+        SQL;
+        $connection = $this->container->get('database_connection');
+        $result = $connection->executeQuery($sql, ['columnName' => $columnName])->fetch();
+        if (!empty($result)) {
```
Comment (grena) [inline upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php:73]: You can directly return `!empty($result)` if you want

**Review by grena — APPROVED** [review]
:+1: 

As there is a DB migration, don't forget to give some information about metrics: 
- what's the max number of job instances in production

File: upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php (hunk 1 above)
Comment (MadeWilson) [reply upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php:73]: your god damn right !

**Review by ghoussard — APPROVED** [review]

**Comment by ghoussard** [comment]
refresh

**Pull request merged**